gensim/models/word2vec.py

```diff
@@ -1195,100 +1195,80 @@ def intersect_word2vec_format(self, fname, lockf=0.0, binary=False, encoding='ut
 
     def most_similar(self, positive=[], negative=[], topn=10, restrict_vocab=None, indexer=None):
         """
-        Please refer to the documentation for
-        `gensim.models.KeyedVectors.most_similar`
-        This is just a forwarding function.
-        In the future please use the `gensim.models.KeyedVectors` instance in wv
+        Deprecated. Use self.wv.most_similar() instead.
+        Refer to the documentation for `gensim.models.KeyedVectors.most_similar`
         """
 
```

Comment:
One more PEP8/PEP257 nitpick: no blank line after docstring (remove the empty line).